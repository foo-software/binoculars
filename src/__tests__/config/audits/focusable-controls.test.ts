import auditFocusableControls from '../../../config/audits/focusable-controls';

describe('focusable-controls audit', () => {
  it('should match snapshot', () => {
    expect(auditFocusableControls('en')).toMatchSnapshot();
  });
});
