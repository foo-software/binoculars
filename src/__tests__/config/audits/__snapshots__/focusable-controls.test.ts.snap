// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`focusable-controls audit should match snapshot 1`] = `
Object {
  "description": "Custom interactive controls are keyboard focusable and display a focus indicator. [Learn more](https://web.dev/focusable-controls/).",
  "title": "Interactive controls are keyboard focusable",
}
`;
